CMakeLists.txt

```diff
@@ -208,6 +208,11 @@ zephyr_compile_options(
   ${TOOLCHAIN_C_FLAGS}
 )
 
+# @Intent: Enforce standard type correspondance to match Zephyr usage.
+zephyr_compile_options(
+  --imacros=${ZEPHYR_BASE}/include/toolchain/zephyr_stdint.h
```

Comment:
Was not aware --flag=value was supported for imacros. This simplifies things.

Nice find.